src/components/Home.tsx

```diff
@@ -188,6 +198,15 @@ const Home: React.FC = () => {
                     </Link>
                 </div>
             </div>
+            <div className="columns is-mobile is-vcentered">
+                <span className="column is-4 has-text-weight-bold">Title</span>
+                <TextBox
+                    className="column pt-0"
+                    name="title"
+                    updateValue={updateTitle}
+                    value={currentValue.title}
+                />
+            </div>
```

Comment:
The reason I don't have this is because I want to make it as easy as possible. If you want this I don't think this should be the first field to fill in.

Reply:
I think you're right.